content/community-targets/NESHTEC-NeshNode.md

```diff
@@ -0,0 +1,96 @@
+# NESHTEC NeshNode
+
+NeshNode is a flexible, high performance board featuring a powerful STM32F7 dual core RISC based microcontroller, on-board ethernet, battery charger, lots of I/Os, two MikroE® Click™ sockets and a lot of tiny details, like a robust USB-C connector.
+
+Furthermore, our board has been extensively tested by the independent testing institute [TÜV Nord](https://www.tuev-nord.de/) for EMC/EMI (electromagnetic compatibility / interference), which means that it is fully compliant with EU regulations (CE).
+
+![NESHTEC NeshNode](../../images/community-targets/NESHTEC-NeshNode-STM32F7-1.1-nf.png)
+
```

Comment:
what is needed to have .NET nanoFramework deployed on it? Any specific image? Is all supported? Any specific board package that you can provide to accelerate developer development for nanoFramework?
See https://github.com/nanoframework/nf-Community-Targets, all the community targets have specific code on top to improve the experience. See as a simple example those helpers: https://github.com/nanoframework/nf-Community-Targets/blob/main/ChibiOS/PybStick2x/managed_helpers/PybStick2x.cs

Reply:
I'm to add a community target for this board.
It's OK to add the markdown labels for the firmware as the others have.
Name will be following the usual pattern: `NESHTEC_NESHNODE`

Reply:
Awesome, thank you for all your support